bc_obps/registration/fixtures/mock/contact.json

```diff
@@ -234,19 +262,5 @@
       "phone_number": "+16044011249",
       "operator_id": "4a792f0f-cf9d-48c8-9a95-f504c5f84b12"
     }
-  },
-  {
-    "model": "registration.contact",
```

Comment:
Papa Enterprises doesn't have a user_operator record, so it shouldn't have any contacts